iOS Example Frame/iOS Example Frame/Storyboards/ar-001.lproj/Main.strings

```diff
@@ -0,0 +1,48 @@
+
+/* Class = "UILabel"; text = "Mobile SDK"; ObjectID = "60b-6U-Yk7"; */
+"60b-6U-Yk7.text" = "Mobile SDK";
```

Comment:
Do we have to localise the demo app as well, in order for the SDK localisations to surface?

From my understanding our localisation is for SDK. Demo app doesn't need localisations (or even as in this case localisation declarations with English content) ?

Reply:
This ticket is only for Frames. We can introduce it also for example app. @patrick-hoban-cko what do you think?

Reply:
The reason for my question is to check whether we should just not commit these files in the demo app. Had a play around locally and I seem to get the arabic text shown in SDK screens even if demo project doesn't declare arabic localisation, which imo would strip unnecessary distractions from the demo implementation.

Reply:
I don't think we need localisations for the demo app in the SDK, as that'd be adding things into the SDK that shouldn't be there